src/Utilities/Idm/mf6blockfile/StressGridInput.f90

```diff
@@ -249,19 +248,18 @@ subroutine ingrid_reset(this)
                                   this%mf6_input%mempath, this%sourcename)
     end if
     !
-    ! -- reset input context memory for parameters
     do n = 1, this%nparam
       if (this%param_reads(n)%invar /= 0) then
-        !
-        ! -- set definition
-        idt => this%mf6_input%param_dfns(this%idt_idxs(n))
         !
         ! -- reset read state
         this%param_reads(n)%invar = 0
-
+        !
       end if
     end do
     !
+    ! -- explicitly reset auxvar array each period
+    this%bndctx%auxvar = DZERO
```

Comment:
We should probably loop through each array element and set to zero, rather than using a single expression.  Maybe it doesn't matter with latest compilers, but these types of assignments have given us stack issues in the past.